integrations/jina.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: integration
+name: Jina
+description: Use Jina embedding models with Haystack
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: deepset_ai
+        linkedin: deepset-ai
+pypi: https://pypi.org/project/jina-haystack/
+repo: https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/jina
+type: Model Provider
+report_issue: https://github.com/deepset-ai/haystack-core-integrations/issues
+logo: /logos/jina.png
+version: Haystack 2.0
+toc: true
+---
+
+### **Table of Contents**
+
+- [Haystack 2.0](#haystack-20)
+  - [Installation](#installation)
+  - [Usage](#usage)
+
+## Haystack 2.0
+
+You can use [Jina embedding Models](https://jina.ai/embeddings) in your Haystack 2.0 pipelines with the Jina [Embedders](https://docs.haystack.deepset.ai/v2.0/docs/embedders).
+
+### Installation
+
+```bash
+pip install jina-haystack
+```
+
+### Usage
+
+You can use Jina Embedding models with two components: [JinaTextEmbedder](https://github.com/deepset-ai/haystack-core-integrations/blob/main/integrations/jina/src/jina_haystack/text_embedder.py) and [CohereDocumentEmbedder](https://github.com/deepset-ai/haystack-core-integrations/blob/main/integrations/jina/src/jina_haystack/document_embedder.py).
```

Comment:
I think `CohereDocumentEmbedder` should be `JinaDocumentEmbedder`

Reply:
Thanks a lot, Changed